### PR TITLE
Collision avoidance tutorial fix

### PR DESCRIPTION
**Pull request by hello-adamc**
Edited tutorial to include fix related to where the custom collision model module is created. This is to avoid import errors.

### CODE REVIEW

File: docs/collision_avoidance_tutorial.md [hunk 1]
```diff
@@ -194,6 +201,19 @@ collision_arm_table:
   py_class_name: 'CollisionArmTable'
   py_module_name: 'collision_arm_table'
 
+```
+However if we create our custom class within the 'stretch_body' directoy the yaml file should be configured as follows:
```
Comment (hello-binit) [inline docs/collision_avoidance_tutorial.md:205]: typo

File: docs/collision_avoidance_tutorial.md [hunk 2]
```diff
@@ -194,6 +201,19 @@ collision_arm_table:
...
+
+```yaml
+robot_collision:
+  models:
+  - collision_arm_table
+
+collision_arm_table:
+  enabled: 1
+  py_class_name: 'CollisionArmTable'
+  py_module_name: 'stretch_body.collision_arm_table'
```
Comment (hello-binit) [inline docs/collision_avoidance_tutorial.md:215]: Let's consolidate these two. I think we should recommend they do not create the file in the stretch_body directory (so the export command becomes necessary).

**Review by hello-binit — CHANGES_REQUESTED** [review]

File: docs/collision_avoidance_tutorial.md (hunk 2 above)
Comment (hello-adamc) [reply docs/collision_avoidance_tutorial.md:215]: Will do. I will remove this block and the text above and just include the explanation of the bash command.

**Review by hello-binit — APPROVED** [review]
Looks good!

**Pull request merged**